spec/spec_helper.rb

```diff
@@ -1,7 +1,24 @@
 require "simplecov"
-require 'coveralls'
 
-SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+if ENV["LCOV_REPORT_PATH"]
+  require "simplecov-lcov"
+
+  SimpleCov::Formatter::LcovFormatter.config do |c|
+    c.report_with_single_file = true
+    root_dir = File.expand_path('..', __dir__)
+    c.single_report_path = File.expand_path(ENV.fetch('LCOV_REPORT_PATH'), root_dir)
+  end
+
+  SimpleCov.formatter = SimpleCov::Formatter::LcovFormatter
+
+  # HACK: https://github.com/fortissimo1997/simplecov-lcov/pull/25
```

Comment:
Instead of this hack - just specify a version constraint to SimpleCov `=>0.18`. Wouldn't that solve this issue?

Reply:
simplecov-0.18.0 supports only ruby >= 2.4, so it gonna be some another hack to make it work.

Reply:
Let's just bump minimum Ruby version to >= 2.4 or does it need some extra effort? I guess you didn't mean this possibility to a "hack".

Reply:
Sorry for such delay, was quite buy with other stuff. Bumped ruby requirement to `>= 2.4.0`. 
It would require some changes to upcoming PRs, but overall will simplify some stuff.